.vsts-dotnet.yml

```diff
@@ -47,7 +47,7 @@ stages:
     - name: TeamName
       value: MSBuild
     - name: VisualStudio.MajorVersion
-      value: 16
+      value: 17
```

Comment:
This PR is good to go if https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/324256 passes perf DDRITs.

Reply:
Looks like it passed.